images/minimal-notebook/setup-scripts/setup_julia.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+# Requirements:
+# - Run as the root user
+# - The JULIA_PKGDIR environment variable is set
+
+import os
+import platform
+import shutil
+import subprocess
+from pathlib import Path
+
+import requests
+
+
+def unify_aarch64(platform: str) -> str:
```

Comment:
One missing docstring

Reply:
Thanks, fixed it. I copy-pasted this function from another place, so fixed it there as well.
This is the copy-paste I can live with 😆